mirdata/datasets/compmusic_jingju_acappella.py

```diff
@@ -0,0 +1,399 @@
+
+    return annotations.EventData(np.array([start_times, end_times]).T, events)
+
+
+@io.coerce_to_string_io
+def load_phrases(fhandle: TextIO) -> annotations.LyricData:
+    """Load phrases in western characters
+
+    Args:
+        fhandle (str or file-like): path or file-like object pointing to a lyric annotation file
+
+    Returns:
+        LyricData: lyric phrase annotation
+
+    """
+    start_times = []
+    end_times = []
+    lyrics = []
+
+    reader = csv.reader(fhandle, delimiter="\t")
+    for line in reader:
+        start_times.append(float(line[0]))
+        end_times.append(float(line[1]))
+        lyrics.append(line[2] if line[2] != "sil" else "")
+
+    return annotations.LyricData(
+        np.array([start_times, end_times]).T,
+        lyrics,
+    )
+
+
+@io.coerce_to_string_io
+def load_phrases_char(fhandle: TextIO) -> annotations.LyricData:
+    """Load phrases in chinese characters
+
+    Args:
+        fhandle (str or file-like): path or file-like object pointing to a lyric annotation file
+
+    Returns:
+        LyricData: lyric phrase annotation
+
+    """
+
+    start_times = []
+    end_times = []
+    lyrics = []
+
+    reader = csv.reader(fhandle, delimiter="\t")
+    for line in reader:
+        start_times.append(float(line[0]))
+        end_times.append(float(line[1]))
+        lyrics.append(line[2] if line[2] != "sil" else "")
+
+    return annotations.LyricData(
+        np.array([start_times, end_times]).T,
+        lyrics,
+    )
+
+
+@io.coerce_to_string_io
+def load_syllable(fhandle: TextIO) -> annotations.EventData:
+    """Load syllable
+
+    Args:
+        fhandle (str or file-like): path or file-like object pointing to a syllable annotation file
+
+    Returns:
+        EventData: syllable annotation
+
+    """
+
+    start_times = []
+    end_times = []
+    events = []
+
+    reader = csv.reader(fhandle, delimiter="\t")
+    for line in reader:
+        start_times.append(float(line[0]))
+        end_times.append(float(line[1]))
+        events.append(line[2] if line[2] != "sil" else "")
+
+    return annotations.EventData(np.array([start_times, end_times]).T, events)
+
+
+@core.docstring_inherit(core.Dataset)
+class Dataset(core.Dataset):
+    """
+    The compmusic_jingju_acappella dataset
+    """
+
+    def __init__(self, data_home=None):
+        super().__init__(
+            data_home,
+            name="compmusic_jingju_acappella",
+            track_class=Track,
+            bibtex=BIBTEX,
+            remotes=REMOTES,
+            license_info=LICENSE_INFO,
+        )
+
+    @core.cached_property
+    def _metadata(self):
+        metadata_path_laosheng = os.path.join(
+            self.data_home,
+            "catalogue - laosheng.csv",
+        )
+        metadata_path_dan = os.path.join(
+            self.data_home,
+            "catalogue - dan.csv",
+        )
+        if not os.path.exists(metadata_path_laosheng) or not os.path.exists(
```

Comment:
I guess this one is ok, maybe just double check and add parenthesis if needed

Reply:
I have split the condition into two, so we avoid ambiguities and we specify to the user which of the two metadata files are missing (in case only one of the two is not there).